api-reference/api-reference-template.md

```diff
@@ -0,0 +1,204 @@
+# API Reference
+
+{Before using this template, read [the accompanying guide](api-reference-guide.md) to this template}.
+
+
+## Overview
+
+This reference describes the Application Program Interfaces (APIs) that help you {access | customize | program} {product}'s {features | functionality}.
+
+### Base URL
+
+The APIs referenced in this documentation use the following base URL:
+
+```
+https://api.example.com
+```
+
+
+### Authorization
+
+All requests to the API must be authenticated and authorized before they can access or manage resources on the {environment | platform}.
+
+The {product} API uses the `{auth_type}` authorization type.
+
+You can authenticate your requests by using {authentication type and method}.
+
+For example:
+
+```
+{EXAMPLE}
+```
+
+
+API requests without valid authentication will fail with a status of `{error_code}`.
```

Comment:
I would probably just cover this in an errors section, if at all. REST API unauthorized requests have a standard response HTTP status code of 401, and API documentation doesn't typically cover standard error codes. You would document the error only if it is non-standard in some way, such as if it returns a custom payload or uses a non-standard HTTP code.